eZ/Publish/Core/Repository/Values/ObjectState/ObjectState.php

```diff
@@ -26,16 +23,33 @@
  */
 class ObjectState extends APIObjectState
 {
-    use MultiLanguageTrait;
-    use MultiLanguageNameTrait;
-    use MultiLanguageDescriptionTrait;
```

Comment:
> What about instead deprecating property 'defaultLanguageCode', making it available via magic __get() for BC, and switch over to using 'mainLanguageCode' even for obj states and stategroups ?

Done in: 767ac9c followed by REST-related updates in b6a6f84 and 41538a2.
I kept `defaultLanguageCode` on [Create and Update Structs](https://github.com/ezsystems/ezpublish-kernel/tree/master/eZ/Publish/API/Repository/Values/ObjectState) as this seems to me like out of scope, but if you wish I could do that as well.
// cc @andrerom